docs/src/main/mdoc/typeclasses/order1.md

```diff
@@ -0,0 +1,247 @@
+  Eq.instance{
+    case (x, y) =>
+      x.name === y.name &&
+      x.age === y.age &&
+      x.height === y.height
+  }
+```
+
+The reason we have to do this is that we require an `Eq` for _each_ concrete type `F[A]` for each possible `F[A]` in our data type. An alternative approach might be to just define `Eq` instances for the types of `F` we expect to use.
+
+```scala
+def eqInstanceForPersonOption: Eq[Person[Option]] = ???
+def eqInstanceForPersonId: Eq[Person[Id]] = ???
+```
+
+This is better in that we don't have to express N number of constraints for each permutation of `F[A]`, but restricts our instances in ways we would rather avoid. Who is to say that we won't want to use `Either[Throwable, *]` in the future as our `F[_]` value.
+
+However if we demand that our `F` has an `Eq1` instance, the situation improves quite a bit.
+
+```scala mdoc
+import cats._
+import cats.syntax.all._
+
+implicit def eqInstanceForPerson[F[_]: Eq1]: Eq[Person[F]] =
+  Eq.instance{
+    case (x, y) =>
+      x.name === y.name &&
+      x.age === y.age &&
+      x.height === y.height
+  }
+```
+
+Recall that `Eq1` can derive an `Eq[F[A]]` for any `A` which has an `Eq`. In this way our constraint of `Eq1[F]` is actually more precise to what we are trying to express. We already know that `String`, `Int`, and `Double` have `Eq` instances, what we really want to express is that the instance is valid for any `F[_]` such that we can create an `Eq[F[A]]` if we already know how to compare values of type `A` and this is exactly what `Eq1` let's us do.
+
+# Order1
+
+Now that we've looked at how `Eq1` can reduce the number of constraints required to write instances for higher kinded data, let's examine how it can reduce the number _instances_ we need to write for many `F` types.
+
+As a motivating example, let us consider writing an `Order` instance for `Option`.
+
+```scala mdoc:reset
+import cats._
+import cats.syntax.all._
+
+def orderInstanceForOption[A: Order]: Order[Option[A]] =
+  Order.from{
+    case (Some(x), Some(y)) =>
+      x.compare(y)
+    case (None, None) =>
+      0
+    case (Some(_), _) =>
+      1
+    case _ =>
+      -1
+  }
+```
+
+At first glance this seems perfectly straight forward and fine. Now let's try to use this instance to get an `Eq` for a few different types of `Option`.
+
+```scala mdoc
+(orderInstanceForOption[Int]: Eq[Option[Int]])
+```
+
+That works just fine.
+
+```scala mdoc:fail
+(orderInstanceForOption[Map[String, String]]: Eq[Option[Map[String, String]]])
+```
+
+This fails, because while `Map[String, String]` _does_ have an `Eq` instance, it doesn't have an `Order` instance but our instance demands that `A: Order`. Classically the solution to this problem one would use Scala's lower priority implicit behavior to provide multiple instances, in this case we'd need to provide one for `PartialOrder` and `Eq`, having 3 different implicit scopes.
```

Comment:
Are the 3 scopes actually needed here? Order[Option[Int]] <: Eq[Option[Int]] so if both are in the same implicit scope, the compiler will pick the one with the most precise type.

Reply:
No idea.

Reply:
I'll update that writeup shortly. Thanks for the insight!

Reply:
updated